common/artifacts/000_wc_artifacts.txt

```diff
@@ -305,8 +318,8 @@ xalatath = {
 	martial = 1
 	combat_rating = 20
 
-	monthly_character_prestige = 1
-	monthly_character_piety = 0.5
+	monthly_character_prestige = 8
```

Comment:
what

Reply:
I did change that? Oh, must be typo. Probably meant 0.8. I'll check later at home...

Reply:
fixed